custom/git-app-deployer/main.go

```diff
@@ -159,16 +166,14 @@ func CreateApplication(appVars *AppVars, delay int, clientset *kubernetes.Client
 		log.Info("[Status]: Namespace created successfully")
 	}
 
-	if err := CreateSockShop("/var/run/"+appVars.filePath, appVars.namespace, appVars.operation); err != nil {
-		log.Errorf("Failed to install sock-shop, err: %v", err)
+	if err := CreateApp("/var/run/"+appVars.filePath, appVars.namespace, appVars.operation); err != nil {
+		log.Infof("Failed to install %s", appVars.namespace)
```

Comment:
Why we convert error to info?

Reply:
We have been handing error on line nom 56.